ddl/column.go

```diff
@@ -326,6 +326,7 @@ func (d *ddl) backfillColumn(t table.Table, columnInfo *model.ColumnInfo, reorgI
 			return errors.Trace(err)
 		}
 
+		job.SetRowCount(int64(count))
```

Comment:
The count of the job may not starts from zero. Set it to local count may not be the total processed count.